tensorflow/lite/kernels/lsh_projection_test.cc

```diff
@@ -98,7 +103,12 @@ TEST(LSHProjectionOpTest2, Sparse1DInputs) {
 
   m.Invoke();
 
+#if defined(__s390x__)
```

Comment:
Wondering if this code has been exercised? Are you able to run all the tests on a big endian machine?

Reply:
Yeah I tried on s390x machine, and this macro is working - I do not have access to other big endian machines so not 100% sure whether their results will be the same as s390x (theoretically it should be, but not tested). If you think it is okay, I could make it more general and apply to all big endian machines.

Reply:
I would recommend making it more generic so that if your change fail on other big endian machine, it will be quickly caught up by simply running all the unit tests.

Reply:
Sounds good, I will make that change, thanks.

Reply:
Sounds good, I will make that change, thanks.